.github/workflows/triton-benchmarks.yml

```
@@ -74,11 +74,13 @@ jobs:
           echo "REPORTS=$PWD/reports" >> $GITHUB_ENV
 
       - name: Install benchmarks
+        id: install
         run: |
           cd benchmarks
           python setup.py install
 
       - name: Run Triton Softmax kernel benchmark
+        if: ${{ steps.install.outcome == 'success' && (success() || failure()) }}
```

Out of curiosity, have you tried
```suggestion
        if: ${{ steps.install.outcome == 'success' && !cancelled() }}
```